### PR TITLE
Add parser cpuset cpus

### PR DESCRIPTION
**Pull request by wushiqinlou**

### CODE REVIEW

**Comment by wushiqinlou** [comment]
The JIRA link: https://projects.engineering.redhat.com/browse/INSPEC-78

**Comment by wushiqinlou** [comment]
The new spec: https://github.com/RedHatInsights/insights-frontend-assets/pull/150

File: insights/parsers/cpuset_cpus.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+"""
+InitProcessCgroup - File ``/sys/fs/cgroup/cpuset/cpuset.cpus``
```
Comment (xiangce) [inline insights/parsers/cpuset_cpus.py]: `CpusetCpus` ?

File: insights/parsers/cpuset_cpus.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    """
+
+    def parse_content(self, content):
+        self.data = []
+        self.cpu_number = 0
+        values = content[0].strip().split(",")
+        for value in values:
+            if "-" in value:
+                # Parser the case that the value is like "2-4"
+                start, end = value.split("-")
```
Comment (xiangce) [inline insights/parsers/cpuset_cpus.py]: A simplify way could be:
```
self.data.extend([str(i) for i in range(int(start), int(end)+1)])
```

File: insights/parsers/cpuset_cpus.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+This file shows the default cgroup cpuset.cpu of system. The format
+of the content is string including comma.
+"""
+
+from .. import parser, CommandParser, LegacyItemAccess
+from insights.specs import Specs
+
+
+@parser(Specs.cpuset_cpus)
+class CpusetCpus(CommandParser, LegacyItemAccess):
```
Comment (xiangce) [inline insights/parsers/cpuset_cpus.py]: `LegacyItemAcess` does not work when `self.data` is not a `dict`. Better remove it.

File: insights/parsers/cpuset_cpus.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        cpu_number (int): It is used to display the number of allowed cpu.
+
+    A small sample of the content of this file looks like::
+
+        0,2-4,7
+
+    Examples:
+        >>> type(cpusetinfo)
+        <class 'insights.parsers.cpuset_cpus.CpusetCpus'>
+        >>> cpusetinfo.data
```
Comment (xiangce) [inline insights/parsers/cpuset_cpus.py]: Better rename the `self.data` to a more representative name such as `cpus`, `cpu_set`.

File: insights/parsers/cpuset_cpus.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+
+@parser(Specs.cpuset_cpus)
+class CpusetCpus(CommandParser, LegacyItemAccess):
+    """
+    Class ``CpusetCpus`` parses the content of the ``/sys/fs/cgroup/cpuset/cpuset.cpus``.
+
+    Attributes:
+
+        cpu_number (int): It is used to display the number of allowed cpu.
```
Comment (xiangce) [inline insights/parsers/cpuset_cpus.py]: And the `self.data` (or the new name if you will rename to) should be described here,

File: insights/parsers/cpuset_cpus.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
+"""
+CpuSetCpus - File ``/sys/fs/cgroup/cpuset/cpuset.cpus``
+==============================================================
```
Comment (xiangce) [inline insights/parsers/cpuset_cpus.py]: Better keep the `==` lines in the same length with the title.

File: insights/parsers/cpuset_cpus.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+from insights.specs import Specs
+
+
+@parser(Specs.cpuset_cpus)
+class CpusetCpus(CommandParser):
+    """
+    Class ``CpusetCpus`` parses the content of the ``/sys/fs/cgroup/cpuset/cpuset.cpus``.
+
+    Attributes:
+        cpuset (lis): It is used to show the list of allowed cpu.
```
Comment (xiangce) [inline insights/parsers/cpuset_cpus.py]: `lis`->`list`

**Comment by esakaiev** [comment]
test this please

**Pull request merged**